R/read_company_data.R

```diff
@@ -1,42 +1,25 @@
-#' Read in company finacial data processed from eikon exports and AR master data
-#' that contain information on multiple credit risk inputs and company
-#' production plans, aggregated to the ticker/technology/year level for
-#' `asset_type` bonds and aggregated to the company/technology/year level for
-#' `asset_type` equity and aggregated to the loan_taker/technology/year level
-#' for `àsset_type` loans
+#' Read in company financial data processed from eikon exports and AR master data
```

Comment:
please rename file (git mv old new if u want to be facny)

Reply:
done